src/Scrumptious.Data/Models/scrumptiousdbContext.cs

```diff
@@ -0,0 +1,161 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata;
+
+namespace Scrumptious.Data.Models
+{
+    public partial class scrumptiousdbContext : DbContext
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Rename class 'scrumptiousdbContext' to match camel case naming rules, consider using 'ScrumptiousdbContext'. ([csharpsquid:S101](https://sonarcloud.io/organizations/mrodsky-github/rules#rule_key=csharpsquid%3AS101))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=rodskyproj2&pullRequest=2&issues=AWVIzE5BlRPS_ea-RXfi&open=AWVIzE5BlRPS_ea-RXfi)

[ISSUE_KEY]: AWVIzE5BlRPS_ea-RXfi